### DISCUSSION TITLE
Hackathon: 2024-05-28

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#583] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by danielfromearth** [comment]
I worked on #554 — making the formatting consistent with ["Common Changelog"](https://common-changelog.org/), and digging up the past Issues and Pull Requests that correspond to the change entries already present in CHANGELOG.md.
